index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header class="header">
+      <a href="#" id="logo">My logo</a>
+      <div class="nav">
+        <img class="union" src="/images/Union.png" alt="img" />
+      </div>
+    </header>
+
+    <main class="main-section">
+      <h2 class="primary-text">I’m Ankit kush. Glad to see you!</h2>
+      <h2 class="paragraph">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hesitate to contact me.
+      </h2>
+
+      <a href=""> <h2 class="connect">Let's Connect</h2></a>
+
+      <div class="social">
+        <a href="#">
+          <img class="twitter" src="/images/twitter.png" alt="twitter"
+        /></a>
+        <a href="#">
+          <img class="linkedin" src="/images/Linkedin.png" alt="linkedin"
+        /></a>
+        <a href="#">
+          <img class="meta" src="/images/Vector1.png" alt="meta"
+        /></a>
+        <a href="#">
+          <img class="github" src="/images/vector2.png" alt="github"
+        /></a>
+        <a href="#">
+          <img class="vector3" src="/images/vector3.png" alt="twofin"
+        /></a>
+      </div>
```

Comment:
- [ ] Kindly I recommend you to use `ul` tag to group them, and `li` per icon; each icon has a link `a` tag for the social media icons , Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using `href`.